design-documents/graph-ql/coverage/requisitionList.graphqls

```diff
@@ -0,0 +1,261 @@
+## Gift Card product type support will be added after the Product Implementation is developed
+type Query {
+    "The requisitionList query returns the contents of customer's  Requisition List"
+    requisitionList(
```

Comment:
What's the difference between `Query.requisitionList` and `Customer.requisitionList`?

Reply:
@DrewML I created that with Query.requisitionList for fetching a single req list entity and Customer.requisitionLists as list of all req list. Will refactor this with filters added to Customer.requisitionLists